app/cafe.py

```diff
@@ -0,0 +1,20 @@
+import datetime
```

Comment:
Add one blank line between different groups of imports to follow PEP-8 in the future